apis/controller/v1alpha1/devworkspacerouting_types.go

```diff
@@ -205,3 +206,7 @@ type DevWorkspaceRoutingList struct {
 func init() {
 	SchemeBuilder.Register(&DevWorkspaceRouting{}, &DevWorkspaceRoutingList{})
 }
+
+func (d *DevWorkspaceRouting) IsWorkspaceStopped() bool {
+	return d.Annotations != nil && d.Annotations[constants.DevWorkspaceStartedStatusAnnotation] == "false"
+}
```

Comment:
I'm not 100% on whether or not the type definition files should include struct methods like this, but this one is probably fine.